openwisp_monitoring/device/api/views.py

```diff
@@ -57,15 +69,23 @@ class DeviceMetricView(MonitoringApiViewMixin, GenericAPIView):
     """
 
     model = DeviceData
-    queryset = (
-        DeviceData.objects.select_related('devicelocation')
```

Comment:
aren't we using the `devicelocation` data to set the floorplan ID in the metrics?

Reply:
The `devicelocation` data is used in the celery task where the DeviceData object is queried from the database

https://github.com/openwisp/openwisp-monitoring/blob/d4bb639c55cc794d720e2d477ef5683a528fe136/openwisp_monitoring/device/tasks.py#L58-L65

Reply:
ok :+1: